### DISCUSSION TITLE
Why is SetFindTextSelect mode wrong when CompleteEnv.edt is used?

### DISCUSSION THREAD

**Discussion by u-fischer** [WinEdt-Team/WinEdt-Support#54] [1.2 Bugs and Issues]
One of my most used short cut is to type `\end{{` and then a space to close the current environment. Since I updated to Build: 20240513  (v. 11.2) - 64-bit this no longer works as expected: The environment is correctly found closed but the text is marked for copy&paste and as soon as I type the space it disappears again. 

I can't find why this happens. `Text Select` is disabled in the find dialog (and it works fine for a normal search) and in CompleteEnv.edt it is set to zero too: `SetFindTextSelect(0);`  

Any idea what could be the cause?

**Comment by Alex-Simonic** [comment]
This has nothing to do with SetFindTextSelect setting. It is an Active Strings bug that was introduced by accident while fixing the GitHub "bug" report:

https://github.com/WinEdt-Team/WinEdt-Support/discussions/7#discussion-6158343

I will fix this as soon as possible and let you know...

**Comment by Alex-Simonic** [comment]
A new build of WinEdt 11.2 is now available. This problem has been fixed...

**Comment by u-fischer** [comment]
Oh nice. I will test asap!

**Comment by u-fischer** [comment]
Thanks, it works fine again.
